climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -564,6 +564,7 @@ def _cxml_to_df(
             import io
         except ModuleNotFoundError as missing_module:
             LOGGER.exception("%s, Please install it manually", str(missing_module))
+            raise
```

Comment:
@emanuel-schmid I misunderstood the exception logging method - now it's actually reraising the exception like I intended.

Reply:
Closing this thread since it's being addressed in the other.